std_msgs/Header header
float64[36] stiffness_matrix
float64[36] damping_matrix